### DISCUSSION TITLE
Is the javadoc for org.eclipse.m2e.core.embedder.IMaven.createExecutionContext() wrong?

### DISCUSSION THREAD

**Discussion by laeubi** [eclipse-m2e/m2e-core#712] [Q&A]
It says

```
 /**
   * Creates and returns new, possibly nested, maven execution context.
   *
   * @since 1.4
   */
```

but looking at the impl it seems to always create a new `IMavenExecutionContext`, do I miss something?

**Comment by mickaelistria** [comment]
What particularly is wrong? The "possibly nested" part? I think the "possibly" makes that this documentation is correct whatever happens on that topic.

**Reply by laeubi** [reply]
> The "possibly nested" part?

yes, i think if it is not do any nesting we could probably remove/simplfy the javadoc here, I'd like to enhance it a bit and then would simply remove the "possibly nested" part then as it actually never happens.
